### PR TITLE
Add dump to aft

### PR DESCRIPTION
**Pull request by chasehensel**
https://drive.google.com/file/d/1MAParaz21AhW4LA-xm2WsYrsQIAqFmsa/view?usp=sharing

### CODE REVIEW

File: internal/db/schema.go [hunk 1]
```diff
@@ -89,3 +89,7 @@ func (s *Schema) GetEnumValueByID(id ID) (ev EnumValue, err error) {
 	}
 	return &enumValue{storeEnumValue}, nil
 }
+
+func (s *Schema) String() string {
```
Comment (awans) [inline internal/db/schema.go:93]: not sure this should be on schema (vs just DB or Tx)

**Review by awans — CHANGES_REQUESTED** [review]
you're right that this is a bit overwhelming in its current form. 

I can think of two ways to improve it:

1. Parse and format the GUIDs - the keys in Hold are raw bytes, but there's a method in Hold that can convert them back into GUIDs (if you select the right substring). 

2. Add some formatting - new lines and indentation (ie a pretty-print) would go a long way towards making this more usable.

**Comment by chasehensel** [comment]
@awans should be good now

**Review by awans — APPROVED** [review]
LGTM

**Pull request merged**